client/src/setupTests.js

```diff
@@ -1,3 +1,13 @@
-require('jest-localstorage-mock');
 
-window.alert = (message) => {};
```

Comment:
`window.alert` is required to suppress console messages in test output. Needs to be added back.

Reply:
yea i have added back